modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/ComponentMessageProcessorBuilder.java

```diff
@@ -147,4 +147,10 @@ protected ResolverSet getArgumentsResolverSet() throws ConfigurationException {
     return this;
   }
 
+  protected ConfigurationProvider getConfigurationProvider() {
```

Comment:
Could the ConfigurationProvider change between calls?
If not, the result can be cached.